platform/node/src/node_request.hpp

```diff
@@ -11,18 +11,19 @@
 
 namespace node_mbgl {
 
-class NodeMap;
+class NodeRequest;
+
+struct NodeAsyncRequest : public mbgl::AsyncRequest {
+    NodeAsyncRequest();
+    ~NodeAsyncRequest() override;
+    NodeRequest* request;
```

Comment:
nit: default to `nullptr`

Reply:
This is already set to nullptr here - do you want this changed? https://github.com/mapbox/mapbox-gl-native/pull/14847/files/d0f219dde6c20ecea9c4242fe56161550131ff74#diff-9066f832688a89e2efb34836fe4d9083R140